cargo/Cargo.toml

```diff
@@ -8,17 +8,19 @@ license.workspace = true
 
 [dependencies]
 rustsec = { workspace = true }
-clap = { workspace = true, features = ["derive"] }
+clap = { workspace = true, features = ["derive", "wrap_help"] }
 quick-xml = { workspace = true, features = ["serialize"] }
 serde = { workspace = true, features = ["derive"] }
 glob.workspace = true
 tracing-subscriber.workspace = true
 tracing.workspace = true
-tar = "0.4"
 tempfile = "3.8"
 terminfo = "0.8"
 toml = "0.7"
 libroast.workspace = true
+sha3 = { version = "0.11.0-pre.4" }
```

Comment:
Why sha3?

Reply:
ah that's the only thing i first found when i was looking for things to check hashes so i just winged it